tron/kubernetes.py

```diff
@@ -122,136 +125,144 @@ def handle_event(self, event: Event) -> None:
         """
         Transitions Tron's state machine for this task based on events from task_processing.
         """
-        event_id = getattr(event, "task_id", None)
-        if event_id != self.get_kubernetes_id():
-            self.log.warning(
-                f"Event task id (id={event_id}) does not match current task id (id={self.get_kubernetes_id()}), ignoring.",
-            )
-            return
-
-        k8s_type = getattr(event, "platform_type", None)
-        self.log.info(f"Got event for task={event_id} (Kubernetes type={k8s_type}).")
-
         try:
-            self.log_event_info(event=event)
-        except Exception:
-            self.log.exception(f"Unable to log event info for id={event_id}.")
-
-        if k8s_type == "running":
-            self.started()
-        elif k8s_type in KUBERNETES_TERMINAL_TYPES:
-            raw_object = getattr(event, "raw", {}) or {}
-            pod_status = raw_object.get("status", {}) or {}
-            container_statuses = pod_status.get("containerStatuses", []) or []
-            exit_code = 0 if k8s_type == "finished" else exitcode.EXIT_KUBERNETES_ABNORMAL
-
-            if len(container_statuses) > 1 or len(container_statuses) == 0:
-                # shouldn't happen right now, but who knows what future us will do :p
-                self.log.error(
-                    "Got an event for a Pod with zero or multiple containers - not inspecting payload to verify success."
+            # Might be that somewhere in this function, something is throwing an exception and we're not handling it
+            # wrap this whole function in a big block of try/except block
+            # if handle_event errors out for any reason then we will never see this event at all
```

Comment:
small suggestion re: re-wording your debugging notes :)
```suggestion
            # we wrap this entire thing in a try-except as otherwise an error in 
            # logging (which is useful, but not critical) will result in us not
            # processing an event at all (which is critical!)
```